### ISSUE TITLE
Path API deprecation results in genomedata-load error

### ISSUE THREAD

**Issue by annamalkowska** [hoffmangroup/genomedata#70]
Hi, 

Thank you for maintaining the genomedata package! I have an issue with converting bedgraph files into genomedata.

I get the following error:

```shell
genomedata-load --sizes --sequence GRCh38_EBV.chrom.sizes.tsv --track H3K4me1=ENCFF582BHK.bedGraph H3K4me1.genomedata
Error creating genomedata.

Cleanup failed: 'Path' object has no attribute 'isfile'
Traceback (most recent call last):
  File "/home/annam/.local/bin/genomedata-load", line 8, in <module>
    sys.exit(main())
  File "/home/annam/.local/lib/python3.10/site-packages/genomedata/load_genomedata.py", line 317, in main
    load_genomedata(args.gdarchive, tracks, seqfilenames,
  File "/home/annam/.local/lib/python3.10/site-packages/genomedata/load_genomedata.py", line 98, in load_genomedata
    if not Path(seqfilename).isfile():
AttributeError: 'Path' object has no attribute 'isfile'. Did you mean: 'is_file'?

``` 
I have genomedata v1.7.2 and python 3.12.4. I tried both my own files and files downloaded from encode as in the above code. The files are definitely present in the directory. I tried both with specifying the full path and without.  Similar error also happens if I try genomedata-load-seq with just the sizes file: 

```shell
genomedata-load-seq ./gd.genomedata -s ./GRCh38
_EBV.chrom.sizes.tsv
Traceback (most recent call last):
  File "/home/annam/.local/bin/genomedata-load-seq", line 8, in <module>
    sys.exit(main())
  File "/home/annam/.local/lib/python3.10/site-packages/genomedata/_load_seq.py", line 496, in main
    return load_seq(args.gdarchive, args.seqfiles, verbose=args.verbose,
  File "/home/annam/.local/lib/python3.10/site-packages/genomedata/_load_seq.py", line 365, in load_seq
    assert not gdpath.exists()
AssertionError
``` 

Do you have an idea of what could help?

Thank you!

**Comment by EricR86** [comment]
This looks like to be a bug introduced by newer versions of the pypi`path` library as of versions after this past April.
[This is the relevant changelog](https://path.readthedocs.io/en/latest/history.html#v16-12-0).

In the meantime you can downgrade your "path" pypi package to before 16.12. I'm currently testing on 16.7.1.

I will fix this ASAP for a new release. Thanks for the bug report!

**Comment by annamalkowska** [comment]
Thank you so much, I downgraded the path package and it works now!

**Comment by EricR86** [comment]
Fixed in release 1.7.3. Should be fixed on the latest Bioconda release as well.
